src/pages/markets/Markets.tsx

```diff
@@ -28,6 +32,23 @@ const Markets = () => {
 
   useDocumentTitle(stringGetter({ key: STRING_KEYS.MARKETS }));
 
+  const marketsPageBanner = useMemo(() => {
+    // TODO: (TRA-528): Localize text when finallized. Update navigate link to TRUMPWIN-USD (Use Market redux state to ensure existance)
+    if (testFlags.enablePredictionMarketPerp) {
+      return (
+        <$MarketsPageBanner>
+          <span>🇺🇸 Leverage trade the outcome of the U.S. Election</span>
```

Comment:
"leverage trade" as a verb feels a bit weird to me because it feels like "leverage" is a verb and "trade" its object until I read it a few times.

Reply:
I feel like it should be "trade the outcome of the US election with leverage"

Reply:
Lemme track down the final decision on copy. I think we can treat this as a placeholder for now because there is opportunity to change this later when adding to v4-localization.